.circleci/config.yml

```diff
@@ -22,25 +22,26 @@ variables:
   restore-cache: &restore-cache
     restore_cache:
       keys:
-        - v1-dependencies-{{ checksum "requirements.txt" }}
+        - v2-dependencies-{{ checksum "requirements.txt" }}
```

Comment:
Why do we need to increment the deps cache?

Reply:
seemed an easy way to force a cache rebuild, but then realized it was because the mac dependencies cache didn't have a fallback for auto-rebuilding -- can revert if we prefer?

Reply:
Refer would be preferable since this should invalidate the cache when the requirements file changes!

Reply:
we decided to keep the increment, since circleci fails without it

Reply:
yeah there's a messed up cache in there so this is the best solution unless we contact circleci and tell them to purge it.